packages/vue_frontend_rpc/src/assets/css/globalStyles.css

```diff
@@ -253,7 +254,9 @@ div.v-input__slot .v-select__selection {
 }
 
 .main-card {
-    height: 100%
+    height: 100%;
+    box-shadow: none !important;
+    background-color: var(--vscode-editor-background, #ffffff) !important;
```

Comment:
"!important" is not a good practice to use. It is always best to find the right selector and define the css property on it. In this example, you can use: 

```css
.main-card {
    height: 100%;
    box-shadow: none;
    background-color: var(--vscode-editor-background, #ffffff);
}

.v-card--variant-elevated.main-card {
    background-color: var(--vscode-editor-background, #ffffff);
    box-shadow: none;
}
```

Please go over all "!important" you used and use a precise selector instead.